### DISCUSSION TITLE
autheila unable to reach gmail SMTP Service.

### DISCUSSION THREAD

**Discussion by sebasdt** [authelia/authelia#5599] [Q&A] [answered]
hi all!

To jump right into the issue, My uptime-kuma gave an alert that something was wrong with Authelia or Traefik.
Digging a little deeper it seems Authelia and gmail smtp aint working anymore while it did work before.

Here below are the error logs from Authelia with log level set to debug.
```
Authelia | time="2023-06-23T21:24:15+02:00" level=info msg="Authelia v4.37.5 is starting"
Authelia | time="2023-06-23T21:24:15+02:00" level=info msg="Log severity set to debug"
Authelia | time="2023-06-23T21:24:15+02:00" level=info msg="Storage schema is being checked for updates"
Authelia | time="2023-06-23T21:24:15+02:00" level=info msg="Storage schema is already up to date"
Authelia | time="2023-06-23T21:24:15+02:00" level=debug msg="Notifier SMTP client attempting connection to smtp.gmail.com:587"
Authelia | time="2023-06-23T21:24:20+02:00" level=error msg="Failure running the notification provider startup check: error dialing the SMTP server: dial tcp 142.250.27.109:587: i/o timeout" stack="github.com/authelia/authelia/v4/internal/commands/root.go:293 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:87  cmdRootRun\ngithub.com/spf13/cobra@v1.6.1/command.go:920                  (*Command).execute\ngithub.com/spf13/cobra@v1.6.1/command.go:1044                 (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.6.1/command.go:968                  main\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_arm64.s:1172                                      goexit"
authelia  | time="2023-06-23T21:24:20+02:00" level=fatal msg="The following providers had fatal failures during startup: notification" stack="github.com/authelia/authelia/v4/internal/commands/root.go:309 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:87  cmdRootRun\ngithub.com/spf13/cobra@v1.6.1/command.go:920                  (*Command).execute\ngithub.com/spf13/cobra@v1.6.1/command.go:1044                 (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.6.1/command.go:968                  main\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_arm64.s:1172                                      goexit"
authelia exited with code 0
``` 

Config snippet for notifier. I don't see anything wrong here even Dubble checked if the app pass is correct.
(btw This exact snippet example is also shown in the [docs](https://www.authelia.com/configuration/notifications/smtp/#options))
```
notifier:
  smtp:
    username: [Gmail Email]
    # Password can also be set using a secret: https://www.authelia.com/configuration/methods/secrets/
    password:  [working Google app password]
    sender: admin@authelia.[my-domain].com
    host: smtp.gmail.com
    port: 587
```

Anyone expiring the same issue or is there something wrong with my setup?
Thank you for your time

**Comment by james-d-elliott** [comment]
Looks like some kind of network issue per the i/o timeout.

**Reply by sebasdt** [reply]
How be? 
Other containers that are on the same host have a network connection.

Recreating/ fully creating a new container doesn't solve the issue.

**Reply by james-d-elliott** [reply, marked as answer]
When I said network issue I was not saying it didn't have a network connection, there was an i/o timeout when trying to connect to 142.250.27.109 (resolved by DNS from smtp.gmail.com) on port 587. This  could be lack of network connection, a firewall, invalid routes, transient internet issues either at the network border or beyond, your ISP blocking mail connections, network issues at Google, Google blocking you, etc.

You can try providing trace logs, or doing some troubleshooting of why a SMTP connection from your host to that IP and port combination is failing, etc.

**Reply by sebasdt** [reply]
okay, It seems like Gmail was having issues! Just recreated Authelia and its up happely sending emails.
Looked through my firewall logs and didn't see anything about blocking gmail smtp port.
